config/testgrids/conformance/conformance-all.yaml

```diff
@@ -388,14 +388,6 @@ test_groups:
 - name: cel-go-conformance-tests
   gcs_prefix: cel-conformance/test-logs
 
-#ibm ppc64le test results
-- name: ppc64le-conformance-containerd
-  gcs_prefix: ppc64le-kubernetes/logs/periodic-kubernetes-containerd-conformance-test-ppc64le
-  days_of_results: 7
-  column_header:
-  - configuration_value: k8s-build-version
-- name: ppc64le-unit
-  gcs_prefix: ppc64le-kubernetes/logs/periodic-kubernetes-unit-test-ppc64le
```

Comment:
Lets not remove this from the conformance-all

Reply:
`test_groups` defined in one yaml under testgrids folder can be used in multiple dashboard tabs.
As we earlier defined test_group detail only in `config/testgrids/conformance/conformance-all.yaml` but mentioned this in dashboard details in `config/testgrids/kubernetes/sig-node/config.yaml ` (https://github.com/kubernetes/test-infra/blob/master/testgrid/config.md#test-groups)

As we have now defined `test_group` details in `config/testgrids/kubernetes/sig-cloud-provider/ibm/config.yaml` , I thought we can remove it here.

Reply:
then we are good,